README.md

```diff
@@ -317,17 +227,16 @@ experiment/
     ...
 ```
 
-The `Makefile` (or similar build configuration) should perform any necessary
-compilation or preprocessor steps.
-Each benchmark should expose a function (or method) called `run_iter`
-which is the entry point to the benchmark.
-In the case of compiled languages, it often makes sense to add an extra
-class to an existing benchmark, in order to provide an entry point.
+The `Makefile` should build the VMs and benchmarks needed for the experiment.
+
+Each benchmark should expose a function (or method) called `run_iter` which is
```

Comment:
It's a little more complicated for Java because you can't have functions at the top-level:

https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/iterations_runners/iterations_runner.java#L230-L235

https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/iterations_runners/iterations_runner.java#L273

Reply:
I think Java users will realise that.

Reply:
OK :)